apps/wing-api-checker/.projenrc.ts

```diff
@@ -8,10 +8,10 @@ const project = new typescript.TypeScriptProject({
     "wing-api-check": "bin/wing-api-check",
   },
   license: "MIT",
-  authorName: "Monada",
+  authorName: "Wing Cloud",
```

Comment:
Need to undo this change. The publisher is still Monada to avoid changing the id of the extension.

```suggestion
  authorName: "Monada",
```

Reply:
Can keep the new `authorUrl` or need to revert that as well? Also `authorEmail` must be left unchanged?

Reply:
Also, do we need to keep`Monada` as author/publisher just for the VS Code extension or also for the `package.json` and `projenrc.ts` files?

Reply:
In checking this, the only field that matters is "publisher", all other fields can change. The current state looks good

Reply:
@MarkMcCulloh Another thing: I noticed there were half folders with a `LICENSE` file and another half with a `LICENSE.md` file, so I renamed everything to `LICENSE.md` but then monadabot [recreated the LICENSE file for two folders](https://github.com/winglang/wing/pull/3674/commits/46171b39d43c042e9dab3da01f89c67fd4affc66). Is it possible to fix this behaviour?

Reply:
@MarkMcCulloh Another thing: I noticed there were half folders with a `LICENSE` file and another half with a `LICENSE.md` file, so I renamed everything to `LICENSE.md`, but then monadabot [recreated the LICENSE file for two folders](https://github.com/winglang/wing/pull/3674/commits/46171b39d43c042e9dab3da01f89c67fd4affc66). Is it possible to fix this behaviour?